material_components_ios.bzl

```diff
@@ -95,7 +68,7 @@ def mdc_unit_test_suite(
     name = name,
     deps = deps,
     minimum_os_version = minimum_os_version,
-    runners = ios_runners(),
+    runners = DEFAULT_IOS_RUNNER_TARGETS,
```

Comment:
Love the testing cleanup - can you move it to a separate PR in case we need to roll the component changes back?